shared/django_apps/rollouts/migrations/0001_initial.py

```diff
@@ -0,0 +1,127 @@
+    # --
+    # CREATE TABLE "feature_flag_variants" ("name" varchar(200) NOT NULL PRIMARY KEY, "proportion" decimal NOT NULL, "enabled" bool NOT NULL, "override_owner_ids" integer[] NOT NULL, "override_repo_ids" integer[] NOT NULL, "feature_flag_id" varchar(200) NOT NULL REFERENCES "feature_flags" ("name") DEFERRABLE INITIALLY DEFERRED);
+    # --
+    # -- Create constraint feature_flag_name on model featureflag
+    # --
+    # CREATE TABLE "new__feature_flags" ("name" varchar(200) NOT NULL PRIMARY KEY, "proportion" decimal NOT NULL, "salt" varchar(10000) NOT NULL, CONSTRAINT "feature_flag_name" UNIQUE ("name"));
+    # INSERT INTO "new__feature_flags" ("name", "proportion", "salt") SELECT "name", "proportion", "salt" FROM "feature_flags";
+    # DROP TABLE "feature_flags";
+    # ALTER TABLE "new__feature_flags" RENAME TO "feature_flags";
+    # CREATE INDEX "feature_flag_variants_feature_flag_id_fa3a4c02" ON "feature_flag_variants" ("feature_flag_id");
+    # --
+    # -- Create index feature_fla_feature_15a078_idx on field(s) feature_flag of model featureflagvariant
+    # --
+    # CREATE INDEX "feature_fla_feature_15a078_idx" ON "feature_flag_variants" ("feature_flag_id");
+    # --
+    # -- Create constraint feature_flag_variant_name on model featureflagvariant
+    # --
+    # CREATE TABLE "new__feature_flag_variants" ("name" varchar(200) NOT NULL PRIMARY KEY, "proportion" decimal NOT NULL, "enabled" bool NOT NULL, "override_owner_ids" integer[] NOT NULL, "override_repo_ids" integer[] NOT NULL, "feature_flag_id" varchar(200) NOT NULL REFERENCES "feature_flags" ("name") DEFERRABLE INITIALLY DEFERRED, CONSTRAINT "feature_flag_variant_name" UNIQUE ("name"));
+    # INSERT INTO "new__feature_flag_variants" ("name", "proportion", "enabled", "override_owner_ids", "override_repo_ids", "feature_flag_id") SELECT "name", "proportion", "enabled", "override_owner_ids", "override_repo_ids", "feature_flag_id" FROM "feature_flag_variants";
+    # DROP TABLE "feature_flag_variants";
+    # ALTER TABLE "new__feature_flag_variants" RENAME TO "feature_flag_variants";
+    # CREATE INDEX "feature_flag_variants_feature_flag_id_fa3a4c02" ON "feature_flag_variants" ("feature_flag_id");
+    # CREATE INDEX "feature_fla_feature_15a078_idx" ON "feature_flag_variants" ("feature_flag_id");
+    # COMMIT;
+
+    operations = [
+        migrations.CreateModel(
+            name="FeatureFlag",
+            fields=[
+                (
+                    "name",
+                    models.CharField(max_length=200, primary_key=True, serialize=False),
+                ),
+                (
+                    "proportion",
+                    models.DecimalField(decimal_places=3, default=0, max_digits=4),
+                ),
+                (
+                    "salt",
+                    models.CharField(
+                        default=shared.django_apps.rollouts.models.default_random_salt,
+                        max_length=10000,
```

Comment:
the salt only need to be a few characters to effectively re-randomize the feature variant buckets. you can set this to like 32